src/sonic-build-hooks/scripts/buildinfo_base.sh

```diff
@@ -22,15 +22,19 @@ log_err()
     echo "$1" 1>&2
 }
 
+# Get the real command not hooked by sonic-build-hook package
```

Comment:
And how? I guess you remove some paths (listing them) in $PATH and check the absolute path of the same command again?

Reply:
It excludes the /usr/local/sbin in env PATH.

---
In reply to: [556218199](https://github.com/Azure/sonic-buildimage/pull/6422#discussion_r556218199) [](ancestors = 556218199)